homeassistant/components/rfxtrx/cover.py

```diff
@@ -95,20 +116,42 @@ async def async_added_to_hass(self):
             if old_state is not None:
                 self._state = old_state.state == STATE_OPEN
 
+    @property
+    def current_cover_tilt_position(self):
+        """Return current position of cover tilt.
+
+        None is unknown, 0 is closed, 100 is fully open.
+        """
+        if self._venetian_blind_mode in (
+            CONST_VENETIAN_BLIND_MODE_US,
+            CONST_VENETIAN_BLIND_MODE_EU,
+        ):
+            return 50  # Rfy does not support tilt position. The non-empty value is used to enable tilt support.
+
```

Comment:
Please return None explicitly here.

Reply:
Done.  I guess this was the wrong way to implement tilt support.  Changed the implementation.